CafeRecords/Pages/Index.cshtml.cs

```diff
@@ -24,13 +20,13 @@ public void OnGet()
         {
             List<Owner> cafeOwners = new List<Owner>();
 
-            string CafeJsonString = GetData("https://data.cityofchicago.org/resource/mqmh-p6ud.json");
-            Cafe[] allCafes = Cafe.FromJson(CafeJsonString);
-            ViewData["allCafes"] = allCafes;
+            string cafeJsonString = GetData("https://data.cityofchicago.org/resource/mqmh-p6ud.json");
+            Cafe[] allCafes = Cafe.FromJson(cafeJsonString);
+            
 
-            string OwnerJsonString = GetData("https://data.cityofchicago.org/resource/ezma-pppn.json");
-            Owner[] allOwners = Owner.FromJson(OwnerJsonString);
-            ViewData["allOwners"] = allOwners;
+            string ownerJsonString = GetData("https://data.cityofchicago.org/resource/ezma-pppn.json");
+            Owner[] allOwners = Owner.FromJson(ownerJsonString);
+            
 
```

Comment:
Changed the notation of the variable to Camel Case notation and removed the ViewData Object which is not used for displaying data in html page.